File: project2/Sheep.java
class Sheep extends Animal {
    public Sheep(int x, int y, World w) {
        super(4, 4, x, y, w);
    }
    
    @Override
    public char draw() {
        return 'S';
    }
    
    @Override
    public Animal clone(int x, int y) {
        return new Sheep(x, y, world);
    }
}

File: project3/sheep.py
from animal import Animal

class Sheep(Animal):
    def __init__(self, x, y, world):
        super().__init__(4, 4, x, y, world)
    
    def draw(self):
        return 'S'
    
    def clone(self, x, y):
        return Sheep(x, y, self.world)
